src/gpu_sysman.c

```diff
@@ -119,15 +119,18 @@ typedef struct {
 } gpu_device_t;
 
 typedef enum {
-  OUTPUT_UNSET = 0,
-  OUTPUT_RAW,
-  OUTPUT_DERIVED,
-  OUTPUT_BOTH, /* 3 = 1 | 2 mask */
-  OUTPUT_TYPES
+  OUTPUT_COUNTER = 1,
+  OUTPUT_RATE = 2,
+  OUTPUT_RATIO = 4,
+  OUTPUT_ALL = 7
```

Comment:
In some previous projects I've used (1 << 0) to create the bits. In this case there are only three bits so it doesn't make a huge difference, but when there are >16 bits used it's maybe more readable.

Reply:
Done.